hack/vendor.sh

```diff
@@ -139,7 +139,7 @@ clone git github.com/docker/docker-credential-helpers v0.3.0
 clone git github.com/docker/containerd 0ac3cd1be170d180b2baed755e8f0da547ceb267
 
 # cluster
-clone git github.com/docker/swarmkit 38857c06dafcf939a56d2650d8e0011b5aace384
+clone git github.com/docker/swarmkit 0aaad148d8c0498f1bd39d7471c9ed0e585620c9 https://github.com/aaronlehmann/swarmkit
```

Comment:
Yes. I am waiting for LGTMs before I merge the upstream PRs in swarmkit. They will block other PRs from vendoring swarmkit until this one is merged, so I am waiting until the last minute.